continuous-integration.yml

```diff
@@ -0,0 +1,208 @@
+                Version: 1
+                Provider: CloudFormation
+              InputArtifacts: [Name: Package]
+              Configuration:
+                ActionMode: CHANGE_SET_REPLACE
+                ChangeSetName: StagingChangeSet
+                RoleArn: !ImportValue JavabuilderCloudFormationRole
+                StackName: !Ref StagingSubDomainName
+                TemplatePath: Package::output.yml
+                TemplateConfiguration: Package::configuration-staging.json
+              OutputArtifacts: [Name: StagingChangeSet]
+            - Name: Deploy
+              RunOrder: 2
+              ActionTypeId:
+                Category: Deploy
+                Owner: AWS
+                Version: 1
+                Provider: CloudFormation
+              InputArtifacts: [Name: StagingChangeSet]
+              Configuration:
+                ActionMode: CHANGE_SET_EXECUTE
+                ChangeSetName: StagingChangeSet
+                StackName: !Ref StagingSubDomainName
+        - Name: Production
+          Actions:
+            - Name: Prepare
+              RunOrder: 1
+              ActionTypeId:
+                Category: Deploy
+                Owner: AWS
+                Version: 1
+                Provider: CloudFormation
+              InputArtifacts: [Name: Package]
+              Configuration:
+                ActionMode: CHANGE_SET_REPLACE
+                ChangeSetName: ProdChangeSet
+                RoleArn: !ImportValue JavabuilderCloudFormationRole
+                StackName: !Ref SubDomainName
+                TemplatePath: Package::output.yml
+                TemplateConfiguration: Package::configuration.json
+              OutputArtifacts: [Name: ProdChangeSet]
+            - Name: Approval
+              RunOrder: 2
+              ActionTypeId:
+                Category: Approval
+                Owner: AWS
+                Provider: Manual
+                Version: 1
+              Configuration:
+                CustomData: !Sub "${SubDomainName} build awaiting approval to Production."
+            - Name: Deploy
+              RunOrder: 3
+              ActionTypeId:
+                Category: Deploy
+                Owner: AWS
+                Version: 1
+                Provider: CloudFormation
+              InputArtifacts: [Name: ProdChangeSet]
+              Configuration:
+                ActionMode: CHANGE_SET_EXECUTE
+                ChangeSetName: ProdChangeSet
+                StackName: !Ref SubDomainName
+#  Webhook:
+#    Type: AWS::CodePipeline::Webhook
+#    Properties:
+#      AuthenticationConfiguration:
+#        SecretToken: !Ref AWS::AccountId
+#      Filters:
+#        - JsonPath: "$.ref"
+#          MatchEquals: refs/heads/{Branch}
+#      Authentication: GITHUB_HMAC
+#      TargetPipeline: !Ref Pipeline
+#      TargetAction: Source
+#      Name: javabuilder-webhook
+#      TargetPipelineVersion: !GetAtt Pipeline.Version
+#      RegisterWithThirdParty: true
```

Comment:
stray comment?

Reply:
Oops. This is still a bit of a work in progress. I commented out the GitHub Webhook temporarily to get the rest of the pipeline working. I need to figure out how to get this working.